packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -328,19 +339,43 @@ class _DiffViewState extends State<DiffView> with AutoDisposeMixin {
   }
 
   Widget _buildTreemapAndTableSplitView() {
-    return Split(
-      axis: Axis.vertical,
-      children: [
-        _buildTreemap(),
-        CodeSizeDiffTable(rootNode: diffRoot),
-      ],
-      initialFractions: const [
-        initialFractionForTreemap,
-        initialFractionForTreeTable,
-      ],
+    return OutlineDecoration(
+      child: Column(
+        children: [
+          areaPaneHeader(
+            context,
+            title: _generateDualFileHeaderText(),
+            needsTopBorder: false,
+          ),
+          Expanded(
+            child: Split(
+              axis: Axis.vertical,
+              children: [
+                _buildTreemap(),
+                CodeSizeDiffTable(rootNode: diffRoot),
+              ],
+              initialFractions: const [
+                initialFractionForTreemap,
+                initialFractionForTreeTable,
+              ],
+            ),
+          ),
+        ],
+      ),
     );
   }
 
+  String _generateDualFileHeaderText() {
+    String output = 'Diffing ';
+    output += controller.oldDiffSnapshotJsonFile.value.isApkFile
+        ? 'APKs: '
+        : 'Snapshots: ';
+    output += controller.oldDiffSnapshotJsonFile.value.displayText;
+    output += ' (OLD) vs (NEW) ';
```

Comment:
add additional space between vs and old/new

Reply:
Done